src/core.js

```diff
@@ -1716,6 +1744,12 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
         /* eslint-disable-next-line no-continue */
         continue;
 
+      } else if (i === 'className' || i === 'tableClassName') {
+        setClassName(settings);
```

Comment:
What do you think to cache settings for `className` and `tableClassName` into one variable with unique class names set them out of `for` loop.
I guess we need to apply proper className for the container only quite before next render, don't we?

Reply:
We should check both className in the same loop. So the cache isn't the appropriate solution, I think.